.github/actions/build-custom-image/action.yaml

```diff
@@ -29,10 +31,18 @@ runs:
       TARGET: ${{ inputs.target }}
       IMAGE_NAME: ${{ inputs.image_name }}
       REPO: ${{ inputs.docker_repo }}
+      BUILDER: ${{ inputs.builder }}
```

Comment:
It does not seem used at all. The `--builder` argument is hard-coded to use `container-builder`

According to https://docs.docker.com/build/builders/#selected-builder we can use `BUILDX_BUILDER` to specify a builder. I think using env is easier to understand than `--builder`